src/main/scala/firrtl/Emitter.scala

```diff
@@ -466,8 +475,12 @@ class VerilogEmitter extends Emitter {
           simulate(s.clk, s.en, printf(s.string, s.args), Some("PRINTF_COND"))
           s
         case (s: WDefInstance) =>
+          val (module, params) = moduleMap(s.module) match {
```

Comment:
The DefModule class is not sealed so you shouldn't be matching over it (unless you use a wildcard....)